action/protocol/poll/consortium.go

```diff
@@ -291,7 +294,10 @@ func getContractReaderForGenesisStates(ctx context.Context, sm protocol.StateMan
 			return nil, err
 		}
 
-		res, _, err := evm.SimulateExecution(ctx, sm, addr, ex, getBlockHash)
+		ctx = evm.WithHelperCtx(ctx, evm.HelperContext{
```

Comment:
WithHelperCtx could be called by the caller, then `getBlockHash` argument can be removed

Reply:
added a TODO